src/EventStore.ClusterNode/ClusterNodeOptions.cs

```diff
@@ -210,7 +210,10 @@ public class ClusterNodeOptions : IOptions {
 
 		[ArgDescription(Opts.DisableExternalTcpTlsDescr, Opts.InterfacesGroup)]
 		public bool DisableExternalTcpTls { get; set; }
-		
+
+		[ArgDescription(Opts.DisableHttpsDescr, Opts.InterfacesGroup)]
+		public bool DisableHttps { get; set; }
```

Comment:
Noted, we could require certificates only if at least one interface requiring TLS is running instead of just checking the options. I'll address this in a separate commit

Reply:
Ok, i've addressed these issues in this commit: 7972e059a07b8c275f3b8cf90d7279c0475fdb30